tests/sdk-consumer/app.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+use \SwedbankPay\Api\Client;
+
+$client = new \SwedbankPay\Api\Client\Client();
+$version = new \SwedbankPay\Api\Client\Version();
+
+var_dump($version->getVersion());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function var_dump() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5303907)